oracle/oracle-how-to/use-bastion-to-access-VM.rst

```diff
@@ -91,45 +137,155 @@ When the bastion plugin shows up as *Running* in the cloud console, you should s
     2023/12/29 18:30:48.011180 plugin.go:52: creating plugin:[bastions], elevated: false, runas: , exe: /var/snap/oracle-cloud-agent/common/bastions
     2023/12/29 18:30:48.203266 health.go:145: started plugin:[bastions]
 
-Now to create a bastion, go to :guilabel:`Identity & Security` > :guilabel:`Bastion` and select :guilabel:`Create bastion`. Provide a name and select the network and subnet where the bastion should be created.
 
-Under ``CIDR block allowlist``, specify the network range from which you want to provide access to your bastion. For instance, you can restrict the bastion access to only valid IP addresses that your ISP assigns to you, so that it would be accessible only from your network. Note that currently, only IPv4 ranges are allowed. 
+You can create the bastion using either the console or the CLI.
+
+.. tabs::
+
+    .. group-tab:: Using console
+
+        Go to :guilabel:`Identity & Security` > :guilabel:`Bastion` and select :guilabel:`Create bastion`. Provide a name and select the network and subnet where the bastion should be created.
+
+        Under ``CIDR block allowlist``, specify the network range from which you want to provide access to your bastion. For instance, you can restrict the bastion access to only valid IP addresses that your ISP assigns to you, so that it would be accessible only from your network. Note that currently, only IPv4 ranges are allowed.
```

Comment:
"network range" should probably be "network address range" or "CIDR range", if I'm correct in guessing that the input here is standard CIDR block notation